src/common/lexical-syntax.k

```diff
@@ -0,0 +1,98 @@
+module LEXICAL-SYNTAX
+//@ \subsection{Comments.sdf}
+//no need to define because K provides fixed layout
+
+//@ \subsection{Identifiers.sdf}
+syntax Id ::= ID [klabel('Id)]
+syntax ID ::= Token{[A-Za-z\_\$][A-Za-z0-9\_\$]*} [onlyLabel]
+
+//@ \subsection{KeyWord.sdf(not needed?)}
+/*
+syntax Keyword ::= "abstract"
+                 | "assert"
+                 | "boolean"
+                 | "break"
+                 | "byte"          
+                 | "case"          
+                 | "catch"         
+                 | "char"          
+                 | "class"         
+                 | "const"         
+                 | "continue"      
+                 | "default"       
+                 | "do"            
+                 | "double"        
+                 | "else"          
+                 | "enum"          
+                 | "extends"       
+                 | "final"         
+                 | "finally"       
+                 | "float"         
+                 | "for"           
+                 | "goto"          
+                 | "if"            
+                 | "implements"    
+                 | "import"        
+                 | "instanceof"    
+                 | "int"           
+                 | "interface"     
+                 | "long"          
+                 | "native"        
+                 | "new"           
+                 | "package"       
+                 | "private"       
+                 | "protected"     
+                 | "public"        
+                 | "return"        
+                 | "short"         
+                 | "static"        
+                 | "strictfp"      
+                 | "super"         
+                 | "switch"        
+                 | "synchronized"  
+                 | "this"          
+                 | "throw"         
+                 | "throws"        
+                 | "transient"     
+                 | "try"           
+                 | "void"          
+                 | "volatile"      
+                 | "while"
+*/
+
+//@ \subsection{LineTerminators.sdf}
+syntax EndOfFile
```

Comment:
To denote line terminates or file ends. I just copied it from sdf, they are not needed?